### ISSUE TITLE
The console doesn't show html(-like) content correctly.

### ISSUE THREAD

**Issue by 14725** [deadlyjack/Acode#415]
The lagecy console failed to take logs with HTML. Here's an example. 

```js
console.log('<style>*{display:none}</style>');
/* or simply type in '<style>*{display:none}</style>' and press Enter  */
```

and then the page is hidden, while the expection is just a normal log.

P.S. a similar thing happens in eruda.js too, but I don't think it stops being maintained by the author. (Trying inspecting the comment below with Eruda's Element tab will take the console away)
```html
<!-- <style>*{display:none}</style> -->
```

**Comment by deadlyjack** [comment]
Thanks for reporting this. This happens because to log the message, I've used innerHTML which renders that style tag. I've to put `!important` in front of all the rules for console to fix it.

**Comment by 14725** [comment]
Thanks very much, however the key problem is not about the style, but simply rendering HTML. 

Why the console should render HTML rather than showing as texts?

**Comment by deadlyjack** [comment]
Correct! I should encode HTML entities before appending it to log. Thanks for the advice.

**Comment by deadlyjack** [comment]
Fixed in version ^1.6.0.
